timemachine/fe/rbfe.py

```diff
@@ -848,7 +857,12 @@ def run_complex(
     )
     complex_box += np.diag([0.1, 0.1, 0.1])  # remove any possible clashes, deboggle later
     complex_host_config = HostConfig(complex_sys, complex_conf, complex_box, nwa)
-    complex_res = estimate_relative_free_energy_bisection(
+    estimate_fxn = (
+        estimate_relative_free_energy_bisection_hrex
+        if md_params.hrex_n_frames_per_iter
+        else estimate_relative_free_energy_bisection
+    )
```

Comment:
nit: would it be worth adding a generic `estimate_relative_free_energy` entry point, to avoid needing to replicate this logic in each `run_leg` function?